tasks/service-worker.js

```diff
@@ -30,6 +30,10 @@ gulp.task('service-worker', callback => {
  *
  */
 gulp.task('service-worker:write', () => {
+    const importScripts = config.sw.importScripts.concat(
```

Comment:
Where does this service worker get generated? (and what part of the site does it handle?)

Reply:
Will be generated in https://www.just-eat.co.uk/service-worker-gw.js and handle new braze implementation

Reply:
Is this a UK specific service worker then?  Reason I ask is that the gulp tasks need to be kept generic and have platform specific things imported when the build is called (as otherwise this service-worker will be imported everywhere that the gulp tasks are used, which may not be what we want?).